service/controller/cluster_resource_set.go

```diff
@@ -44,18 +43,20 @@ import (
 	"github.com/giantswarm/cluster-operator/service/internal/basedomain"
 	"github.com/giantswarm/cluster-operator/service/internal/hamaster"
 	"github.com/giantswarm/cluster-operator/service/internal/podcidr"
+	"github.com/giantswarm/cluster-operator/service/internal/releaseversion"
```

Comment:
This needs merging with master since the resource sets got removed when moving to `operatorkit@v1.0.0`.